code/modules/spells/roguetown/spells5e/cantrips5e.dm

```diff
@@ -938,8 +938,8 @@
 /datum/status_effect/buff/infestation5e/tick()
 	var/mob/living/target = owner
 	var/mob/living/carbon/M = target
-	target.adjustToxLoss(2)
```

Comment:
dont touch infestation that shit already fucks

Reply:
it's literally an irritation at best as it is

Reply:
dropped tox down to 3 at least so it doesn't rock people's shit as hard but yeah 3 per tick that someone could heal off with a potion or antitox is just yuck

Reply:
actually fuckit I reverted infestation for now I might look at it later for a proper rework